### ISSUE TITLE
模型导出

### ISSUE THREAD

**Issue by zenith779** [hotsuyuki/YOLOv5_PyTorch_cpp#1]
请问在导出gpu版本的模型的时候，设置的model.model[-1].export = False吗？

**Comment by hotsuyuki** [comment]
Yes, I set `model.model[-1].export = False` when exporting TorchScript model (in both CPU and GPU).

Reference: https://github.com/yasenh/libtorch-yolov5#torchscript-model-export

**Comment by zenith779** [comment]
> Yes, I set `model.model[-1].export = False` when exporting TorchScript model (in both CPU and GPU).
> 
> Reference: https://github.com/yasenh/libtorch-yolov5#torchscript-model-export

But, when i set `model.model[-1].export = False`,  this get  `TracerWarning:` Converting a ce to be incorrect. We can't record the data flow of Python values, so this value will eans that the trace might not generalize to other inputs!
  if self.grid[i].shape[2:4] != `x[i].shape[2:4]:`, does this affect libtorch load torchscript model?
